src/operators.jl

```diff
@@ -249,21 +268,30 @@ for op in ops_2_to_1_ffz
     SparseConnectivityTracer.is_crossder_zero_global(::T)      = true
 end
 
+is_firstder_arg2_zero_local(::typeof(mod), x, y) = ifelse(y > 0, y > x, x > y)
+is_firstder_arg2_zero_local(::typeof(rem), x, y) = ifelse(y > 0, y > x, x > y)
+
+is_firstder_arg1_zero_local(::typeof(max), x, y) = x < y
+is_firstder_arg2_zero_local(::typeof(max), x, y) = y < x
+
+is_firstder_arg1_zero_local(::typeof(min), x, y) = x > y
+is_firstder_arg2_zero_local(::typeof(min), x, y) = y > x
```

Comment:
I would use wide inequalities

Reply:
What do you mean?

Reply:
`<=`

Reply:
It turns out this is correct to support `max(x, y)` where $x = y$:

```Julia-repl
julia> @test local_jacobian_pattern(x -> max(x[1], x[2]), [1.0, 2.0], G) ≈ [0 1;]
Test Passed

julia> @test local_jacobian_pattern(x -> max(x[1], x[2]), [2.0, 1.0], G) ≈ [1 0;]
Test Passed

julia> @test local_jacobian_pattern(x -> max(x[1], x[2]), [1.0, 1.0], G) ≈ [1 1;]
Test Passed
```
```Julia-repl
julia> @test local_jacobian_pattern(x -> min(x[1], x[2]), [1.0, 2.0], G) ≈ [1 0;]
Test Passed

julia> @test local_jacobian_pattern(x -> min(x[1], x[2]), [2.0, 1.0], G) ≈ [0 1;]
Test Passed

julia> @test local_jacobian_pattern(x -> min(x[1], x[2]), [1.0, 1.0], G) ≈ [1 1;]
Test Passed
```

Reply:
right, because the functions are `iszero`